src/components/Tweet.js

```diff
@@ -1,36 +1,25 @@
-function Tweet() {
+import React from 'react'
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+import Actions from './Actions';
+
+function Tweet(props) {
   return (
     <div className="tweet">
```

Comment:
Here, you should've used the structure the lab provided in the beginning, like:

`<div className="tweet">
      <ProfileImage image = {tweet.user.image}/>

      <div className="body">
        <div className="top">
          <span className="user">
            <User userData={props.tweet.user} />
          </span>

          <Timestamp time={props.tweet.timestamp} />
        </div>

        <Message message={props.tweet.message} />   

        <div className="actions">
        
          <Actions  />
        </div>
      </div>

      <i class="fas fa-ellipsis-h"></i>
    </div>`